docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - PHABRICATOR_UNPRIVILEGED_API_KEY=api-123456789
       - TRANSPLANT_URL=https://stub.transplant.example.com
       - DATABASE_URL=sqlite:////db/sqlite.db
+      - HOST_URL=https://landoapi.mozilla.org
```

Comment:
Agree that we will need this variable. Lets set to `http://lando-api.test` and in ckolos change it for dev, stage, and prod.